README.md

```diff
@@ -183,17 +183,17 @@ You have the option to attach data to the JSON file in all these hooks / steps:
 The only thing you need to provide is the following code in your step files.
 
 ```js
-import cucumberJson from 'wdio-cucumberjs-json-reporter';
+const { attach } = require("wdio-cucumberjs-json-reporter");
```

Comment:
Let's not replace the import example as most people use this. I suggest to add an example of the `require` import under this line.

Note that you are using CJS in your project. Many projects move towards ESM and use imports as shown in this example. We should totally have an example for CJS users too though.